vic/drivers/image/src/vic_image.c

```diff
@@ -108,12 +108,12 @@ main(int    argc,
     // allocate memory
     vic_alloc();
 
-    // allocate memory for routing
-    rout_alloc();   // Routing routine (extension)
-
     // initialize model parameters from parameter files
     vic_image_init();
 
+    // allocate memory for routing
+    rout_alloc();   // Routing routine (extension)
+
     // initialize routing parameters from parameter files
```

Comment:
Omdat domain_struct local_domain pas word geinitializeerd bij vic_init(),
en ik gebruik de waardes local_domain.ncells_active in de code…
On 1 Jul 2016 14:11, "Wietse Franssen" <notifications@github.com> wrote:

> In vic/drivers/image/src/vic_image.c
> <https://github.com/wietsefranssen/VIC/pull/7#discussion_r69290001>:
>
> >      // initialize model parameters from parameter files
> >      vic_image_init();
> >
> > +    // allocate memory for routing
> > +    rout_alloc();   // Routing routine (extension)
> > +
> >      // initialize routing parameters from parameter files
>
> Just curious: waarom heb je rout_alloc verplaatst?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wietsefranssen/VIC/pull/7/files/b067c091c0f2ebcaaaa43aed626afc500bbcd45d#r69290001>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASqP8VSd2MRd4mkn0oYAE_4XH6bV2Ew2ks5qRQPzgaJpZM4JC-E_>
> .
>